lib/events.ts

```diff
@@ -129,10 +130,18 @@ async function sendEvent(balenaUrl: string, event: string, username?: string) {
 		data: Buffer.from(JSON.stringify(trackData)).toString('base64'),
 	};
 	try {
-		await got(url, { searchParams, retry: 0 });
+		await got(url, { searchParams, retry: 0, timeout: 4000 });
 	} catch (e) {
 		if (process.env.DEBUG) {
 			console.error(`[debug] Event tracking error: ${e.message || e}`);
 		}
+		if (e instanceof got.TimeoutError) {
+			throw new AnalyticsRequestTimeout(stripIndent`
+				Timeout submitting analytics event to balenaCloud/openBalena.
+				If you are using the balena CLI in an air-gapped environment without an internet connection,
+				set the BALENARC_OFFLINE_MODE=1 environment variable when using CLI commands that do not strictly
+				require access to balenaCloud, such as balena ssh <ip-addr>.
```

Comment:
Mainly to limit it at 80 characters per line, but also some minor suggestions:
```suggestion
				Timeout submitting analytics event to balenaCloud/openBalena.
				If you are using the balena CLI in an air-gapped environment with a filtered
				internet connection, set the BALENARC_OFFLINE_MODE=1 environment variable
				when using CLI commands that do not strictly require access to balenaCloud.
```